src/js/common/schemaform/FormPage.jsx

```diff
@@ -17,6 +18,15 @@ function focusForm() {
   }
 }
 
+const scroller = Scroll.scroller;
+const scrollToTop = () => {
+  scroller.scrollTo('topScrollElement', window.VetsGov.scroll || {
```

Comment:
So...documentation isn't a silver bullet or anything--especially when you amass too much of it--but I wonder if this is the type of thing that we should have written somewhere.

"`window.VetsGov` is a thing! This is what it does."